lib/open_api_spex/schema.ex

```diff
@@ -470,6 +502,15 @@ defmodule OpenApiSpex.Schema do
       :ok
     end
   end
+  # Note: OpenAPI3's `{"nullable": true}` really means JSON schema's `{}` (i.e. anything)
```

Comment:
Can you clarify this comment?

> Null
> OpenAPI 3.0 does not have an explicit null type as in JSON Schema, but you can use nullable: true to specify that the value may be null.
> The example above may be mapped to the nullable types int? in C# and java.lang.Integer in Java.

My interpretation is that the value must match the schema or be `null`, not anything.

Reply:
Yes this is also my interpretation.
However the schema in `{"nullable": true}` is `{}` which is the schema that matches anything. Do you want me to remove that comment maybe?

I'd be much happier if we'd use `ex_json_schema` instead of rolling out our own schema handling...

Reply:
Ah I get it now after reading https://swagger.io/docs/specification/data-models/data-types/#any. Any type is allowed when no `type` is specified, and `nullable: true` additionally allows `null`.